packages/react-ui/components/Toast/Toast.tsx

```diff
@@ -134,19 +134,13 @@ export class Toast extends React.Component<ToastProps, ToastState> {
         exit={!isTestEnv}
         nodeRef={this.rootRef}
       >
-        <CommonWrapper rootNodeRef={this.setRootRef} {...this.props}>
+        <CommonWrapper rootNodeRef={this.setRootNode} {...this.props}>
```

Comment:
```suggestion
        <CommonWrapper rootNodeRef={mergeRefs([this.setRootNode, this._refToast])} {...this.props}>
```

По духу задачи вроде так надо.